### ISSUE TITLE
random cdt 

### ISSUE THREAD

**Issue by Kirmari** [fireundubh/fallout4#38]
i have seem to get random crash when using auto loot most of them a when i hold my cursor over body and flora

using the latest version

**Comment by Kirmari** [comment]
might as well add that as soon i disable auto loot form load order the crashes stops and the game no longer crashes

**Comment by Swordmage** [comment]
For me, it's coolers. Those little blue chem coolers? If I even so much as accidentally brush a pixel of them, poof. Hard freeze and I have to kill with task manager. Did anyone ever find a way to stop this from happening? It seems to happen only when autoloot is running. I'm thinking it might be UI related? Don't know.

**Comment by Occams-Cat** [comment]
I used to get the crash, but then I started using fireundubh's the "silent loot" mod along with this, for whatever reason, if I was mousing over a container/body etc while this was looting I think the crash was sound related.

**Comment by Kirmari** [comment]
will try that thx

**Comment by Kirmari** [comment]
I have try this out over the last few days and it indeed it seems to be a collision with the sound no crash so far.
many thx to you reply Occams-Cat 

to the author
i suggest implementing silent loot in to auto loot as as fix or state if ppl do have crashes to try silent loot as a fix as least and to be fair i think you own ppl a apology, as it is you mod that causes the crash not the developer side of game i can see why it can get frustrating if you can't replicate the bug you self, to try fix the issue 

**_it shall also be said i love you mods there are awesome_** 

Best Regards
Kirmari

sry for my bad english it is not my natriv

**Comment by Occams-Cat** [comment]
Kirmari, it's not fireundubh's fault for the crashes.
If anything it's the typical bad Bethesda game engine not being able to keep up with the looting sounds and the visuals at the same time.
For what it's worth, I also had to use a mod called "Silent VATS" in order to stop my game from freezing/crashing whenever I used VATS.
So, it's Bethesda's crappy engine.
Personally the Auto-loot and silent-loot mods are best kept separate as it gives people the option to use it or not
